README.md

```diff
@@ -1,14 +1,20 @@
 Conway
+====
 ©1998  Alex Chamberlain
-chambera@cs.pdx.edu
-http://www.cs.pdx.edu/~chambera
+
+Email: chambera@cs.pdx.edu
+
+Website: http://www.cs.pdx.edu/~chambera
```

Comment:
It's dead, Jim.
Since the website 404s, the mail address probably does as well. Seems like Alex managed to graduate some time in the past 20 years... :)
Remove both.

Reply:
@humdingerb I noticed that they didn't work, but I just kept them just to make sure :)